hooks/image/60-kolibri-content

```diff
@@ -4,6 +4,29 @@ if [ "${EIB_IMAGE_KOLIBRI_CONTENT}" != "true" ]; then
   exit 0
 fi
 
+import_kolibri_channel()
+{
+  local channel_id=$1
+  local channel_include_node_ids_var="EIB_KOLIBRI_${channel_id}_INCLUDE_NODE_IDS"
+  local channel_exclude_node_ids_var="EIB_KOLIBRI_${channel_id}_EXCLUDE_NODE_IDS"
```

Comment:
If I'm reading this right, we expect to have a section like this in the config, when selecting subsets of channels (here just `<node_id1>` and `<node_id2>` of `<channel_id1>`, and all but `<node_id5>` from `<channel_id3>`):
```
[kolibri]
install_channels = <channel_id1>
                   <channel_id2>
                   <channel_id3>
<channel_id1>_include_node_ids = <node_id1>
                                 <node_id2>
<channel_id3>_exclude_node_ids = <node_id5>
```
Can we add an example in `default.ini`? Even if just something like this in the comments.

Reply:
This might be reaching the limits of what's reasonably expressable in INI format. Maybe it would be preferable to define the channel selection in a JSON or YAML file in `data/` with the path defined here. Here's some barebones YAML where the top level is a dictionary where keys are the channel IDs and the values are options dictionaries:
```
channel1:
channel2:
  include: node1
channel3:
  exclude: node2
```
That would be pretty easy to parse in python:
```
import yaml
with open('channels.yaml') as f:
    channels = yaml.safe_load(f)
for channel, options in channels.items():
    # Turn None values to empty dicts for simplicity
    options = options or {}
    include_nodes = options.get('include', [])
    exclude_nodes = options.get('exclude', [])
    import_content_cmd = ['kolibri', 'manage', 'importcontent']
    if include_nodes:
        import_content_cmd.append('--node_ids=' + ','.join(include_nodes))
    if exclude_nodes:
        import_content_cmd.append('--exclude_node_ids' + ','.join(include_nodes))
```
And on. In that case I guess I'd split out the importing part of the hook from the kolibri setup part.